bank-service/src/test/java/rs/banka4/bank_service/integration/order/OrderControllerTest.java

```diff
@@ -50,10 +46,16 @@ public class OrderControllerTest {
 
     private UUID stockId;
 
+    private static final String ACCOUNT_NUMBER = DataSourceService.ACCOUNT_JANE_STANDARD_NUMBER;
+
     @BeforeEach
     void setUp() {
+        accountRepo.deleteAll();
         assetRepo.deleteAll();
         listingRepository.deleteAll();
+        actuaryRepository.deleteAll();
```

Comment:
why is this necessary?  the data is cleaned before all functions by `FlywayCleanerExtension` anyway through `DbEnabledTest`